src/System.Net.Sockets/tests/FunctionalTests/System.Net.Sockets.Tests.csproj

```diff
@@ -64,7 +64,15 @@
       <Link>Common\System\Net\Capability.Sockets.cs</Link>
     </Compile>
   </ItemGroup>
-
+  
+    <!-- Windows only tests -->
+  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' ">
+    <Compile Include="SocketOptionNameTest.cs" />
```

Comment:
/cc @pgavlin 
Please add a corresponding *NIX/OSX test or create an issue to ensure that using this flag isn't going to have some unexpected behavior on those platforms.